backend/workflow_manager/workflow_v2/workflow_helper.py

```diff
@@ -575,7 +575,6 @@ def execute_workflow_async(
     @staticmethod
     @shared_task(
         name="async_execute_bin",
-        acks_late=True,
         autoretry_for=(Exception,),
         max_retries=1,
```

Comment:
```suggestion
        max_retries=0,
```
@muhammad-ali-e @ritwik-g shall we also make this change to avoid retrying for all `Exception`? We should make it more nuanced but I don't wish to take up that change with this PR. We can ticket it and tackle that separately